src/stream/KeyExchange.js

```diff
@@ -537,9 +572,9 @@ export function SubscriberKeyExchange(client, { groupKeys = {} } = {}) {
     })
 
     async function getGroupKey(streamMessage) {
```

Comment:
`getGroupKey` definitely has a confusing name now though, since it returns an array of keys, one of which should be the one you're after, or an empty array.